packages/error/src/index.ts

```diff
@@ -53,3 +53,23 @@ export function newMessage(err: any, newMsg: string): ErrorWithData {
   newErr.message = `${newMsg}: ${newErr.message}`;
   return newErr;
 }
+
+export interface AugmentedErrorOptions {
+  status?: number;
+  info?: string;
```

Comment:
Would it be better to have this as `HTMLSafeString | string` so that we can template it into HTML without having to worry about safety? That is, at the moment we presumably need to template it into HTML without further escaping, which is fine if we've already escaped it but not fine if we started with an unsafe string. I'm not sure that's the world's best explanation but I hope you see what I mean 😅

Reply:
I do see what you mean, and if our error page lived in `@prairielearn/html`-land, I'd agree with you. Unfortunately, it's still written in EJS. I like the idea; let's save it for the future?

Reply:
Ok, I see we are using unescaped `<%- error.info %>` on the error page. This feels pretty dangerous but I guess we haven't managed to shoot ourselves in the foot yet with this 😅

Reply:
Well, we did, but I actually changed that here: https://github.com/PrairieLearn/PrairieLearn/pull/8010

Here's a thought: make `status` _only_ accept `HTMLSafeString`, but immediately `toString()` it? Alternatively, I guess we could just unconditionally `toString()` it in the error page? I think either way, we could do better without having to refactor the error page to not use EJS. Let me play around with this.

Reply:
I made this change in f749ba2, it works great!